drainer/executor/mysql.go

```diff
@@ -30,6 +30,15 @@ func newMysql(cfg *DBConfig, sqlMode *string) (Executor, error) {
 }
 
 func (m *mysqlExecutor) Execute(sqls []string, args [][]interface{}, commitTSs []int64, isDDL bool) error {
+	if isDDL {
+		// the index of table is not empty
+		if args[0][1] != "" {
+			schema := args[0][0]
+			useSQL := fmt.Sprintf("use %s;", schema)
```

Comment:
add `use <db>` only if it's not a create table statement like in `GenDDLSQL` before.
or will fail to exec `use <db>` because the downstream has not the <db>

Reply:
revealed